iree/tools/iree-benchmark-module-main.cc

```diff
@@ -219,8 +220,16 @@ int main(int argc, char** argv) {
   // Capture status checking will be delayed at benchmarking runtime.
   auto module_data_status = iree::GetModuleContentsFromFlags(module_data);
   auto prepare_vm_func_status = iree::PrepareIREEVMFunction(
-      instance, device, hal_module, &context, input_module, function, inputs,
-      output_descs, function_name, module_data);
+      &instance, &device, &hal_module, &context, &input_module, function,
+      inputs, output_descs, function_name, module_data);
+  if (prepare_vm_func_status.ok()) {
+    assert(instance);
```

Comment:
Normally we don't use `assert` in IREE codebase; there are utility macros/functions like [`IREE_DCHECK`](https://github.com/google/iree/blob/main/iree/base/logging.h#L328-L343), which gives better error handling/message.

Reply:
I see, thanks! Will send out a follow up to clean it.